cypress/integration/dashboard/create_dashboard_spec.js

```diff
@@ -0,0 +1,36 @@
+describe('Create Dashboard', () => {
+  beforeEach(() => {
+    cy.login();
+    cy.visit('/dashboards');
+  });
+
+  it('create new', () => {
```

Comment:
```suggestion
  it('creates a new dashboard', () => {
```

IMO test names should have readable names, often more verbose, but the idea is that when you read it you get exactly what it asserts only from its name. For E2E tests this is a bit complicated because you actually assert a lot of things in a block to improve performance, but you can read "it creates a new dashboard" :grin:.